conman/routes/managers.py

```diff
@@ -6,6 +6,8 @@
 
 class RouteManager(PolymorphicManager):
     """Helpful methods for working with Routes."""
+    silence_use_for_related_fields_deprecation = True
```

Comment:
Now django < 1.10 is gone, is it worth considering `manager_inheritance_from_future=True`?